### PR TITLE
Chore: Polish Nova Output Page

### LINKED ISSUE #1305: [Task]: Polish Output page
### Task description

- [x] Add mana units to all output mana values
- [x] Add to metadata "Commitment ID" --> `metadata.included.commitmentID` with a link to the right slot (you have in the metadata the slot index too)
- [x] Add link to validator address
- [x] Add link to epoch (unless `0`)
- [x] Add hint to end epoch when its `0` explaining what it means: "When end epoch is 0 it means there is no end epoch defined for this delegation output" (ask chatgpt for a better explanation lol)
- [x] Always display mana, except for the delegation rewards mana which should be displayed only in delegation outputs (but always displayed even when 0)
- [x] Add units to staked amount
- [x] what is `Fixed cost`? Can we explain this number a bit better like..is it a percentage so we should add `%`?

### PR DESCRIPTION
**Pull request by brancoder**
# Description of change

closes #1305 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
The CI is failing 🙏🏼 https://github.com/iotaledger/explorer/actions/runs/8400980359/job/23008866039?pr=1318#step:5:14

File: client/src/helpers/nova/manaUtils.tsx [hunk 1]
```diff
@@ -34,30 +36,48 @@ export function buildManaDetailsForOutput(
...
+        return (
+            <span className="balance-base-token pointer margin-r-5" onClick={() => setIsFormatFull(!isFormatFull)}>
+                {formatAmount(mana ?? 0, manaInfo, isFormatFull)}
+            </span>
+        );
+    };
+
+    const entries = {
         label: "Mana:",
         value: manaDetails?.totalMana,
```
Comment (begonaalvarezd) [inline client/src/helpers/nova/manaUtils.tsx:58]: We should also add units to the toplevel mana

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
Added a mini comment, but everything works very nicely. With the mana units added to the total mana & the CI fixed, we can merge 🌷

**Review by msarcev — APPROVED** [review]

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
The total mana doesnt add up 🙏🏼 
image
output `0xda6e52e4e01393beaf2ccc56a83ca85d63f45dd82274ed4ab17912b7b77a60f8ba8a00000000`

**Comment by msarcev** [comment]
> The total mana doesnt add up 🙏🏼 image output `0xda6e52e4e01393beaf2ccc56a83ca85d63f45dd82274ed4ab17912b7b77a60f8ba8a00000000`

Consequence of typescript being a joke. The types in the source are a "hopefully is this type" at best.

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**